tests/test_collinear_endpoint_cases.py

```diff
@@ -0,0 +1,46 @@
+def test_3():
+    # Test lots of collinear endpoint cases.
+    import random, math, pytest
+    from mt2 import mt2
```

Comment:
Imports should occur at the start of the file, not in this function. Also at most one library per line.

Reply:
Fine, I will fix this.